pkg/kds/mux/server.go

```diff
@@ -39,16 +43,18 @@ func (f OnSessionStartedFunc) OnSessionStarted(session Session) error {
 type server struct {
 	config    multizone.KdsServerConfig
 	callbacks Callbacks
```

Comment:
Shouldn't this be renamed Callback (singular) that's something I should have done.

Reply:
I think it was created with a mind that there might be many functions inside, like OnSessionFinished etc.